aws/rust-runtime/aws-inlineable/Cargo.toml

```diff
@@ -12,7 +12,7 @@ publish = false
 repository = "https://github.com/smithy-lang/smithy-rs"
 
 [features]
-http_1x = ["dep:http-1x", "dep:http-body-1x", "aws-smithy-runtime-api/http-1x"]
+http_1x = ["dep:http-body-1x", "aws-smithy-runtime-api/http-1x"]
```

Comment:
FYI, the feature name was updated to `http-1x` in [the main branch](https://github.com/smithy-lang/smithy-rs/blob/main/aws/rust-runtime/aws-inlineable/Cargo.toml#L15), as the updated nightly compiler told us that those who used this feature were specifying `http-1x` (and we tend to use a hyphen in feature names in our crates).